app/questionnaire/questionnaire_schema.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, questionnaire_json, language_code=DEFAULT_LANGUAGE_CODE):
     def is_hub_enabled(self):
         return self.json.get('hub', {}).get('enabled')
 
+    def get_section_ids_required_for_hub(self):
+        return self.json.get('hub', {}).get('required_completed_sections', [])
```

Comment:
I think that as the second key is within the hub we can be a bit self-referential and go with `requires_completed_sections` - it reads better to me given the tense change

Reply:
I think this the current name is more in line with the rest of the accessors in this file. 'requires' could potentially be construed as being a boolean indicating if it needs completed sections.